src/w3d/math/matrix3.cpp

```diff
@@ -152,18 +144,100 @@ void Matrix3::Multiply(const Matrix3 &A, const Matrix3 &B, Matrix3 *set_res)
     (*set_res)[0][0] = (float)((*Aptr)[0][0] * tmp1 + (*Aptr)[0][1] * tmp2 + (*Aptr)[0][2] * tmp3);
     (*set_res)[1][0] = (float)((*Aptr)[1][0] * tmp1 + (*Aptr)[1][1] * tmp2 + (*Aptr)[1][2] * tmp3);
     (*set_res)[2][0] = (float)((*Aptr)[2][0] * tmp1 + (*Aptr)[2][1] * tmp2 + (*Aptr)[2][2] * tmp3);
-    
+
     tmp1 = B[0][1];
     tmp2 = B[1][1];
     tmp3 = B[2][1];
     (*set_res)[0][1] = (float)((*Aptr)[0][0] * tmp1 + (*Aptr)[0][1] * tmp2 + (*Aptr)[0][2] * tmp3);
     (*set_res)[1][1] = (float)((*Aptr)[1][0] * tmp1 + (*Aptr)[1][1] * tmp2 + (*Aptr)[1][2] * tmp3);
     (*set_res)[2][1] = (float)((*Aptr)[2][0] * tmp1 + (*Aptr)[2][1] * tmp2 + (*Aptr)[2][2] * tmp3);
-    
+
     tmp1 = B[0][2];
     tmp2 = B[1][2];
     tmp3 = B[2][2];
     (*set_res)[0][2] = (float)((*Aptr)[0][0] * tmp1 + (*Aptr)[0][1] * tmp2 + (*Aptr)[0][2] * tmp3);
     (*set_res)[1][2] = (float)((*Aptr)[1][0] * tmp1 + (*Aptr)[1][1] * tmp2 + (*Aptr)[1][2] * tmp3);
     (*set_res)[2][2] = (float)((*Aptr)[2][0] * tmp1 + (*Aptr)[2][1] * tmp2 + (*Aptr)[2][2] * tmp3);
 }
+
+int Matrix3::Is_Orthogonal() const
```

Comment:
This looks like it should be a bool return?

Reply:
Is_Orthogonal is int in the original code so we should do the same.